mon-pix/translations/en.json

```
@@ -361,13 +361,13 @@
                     "status": {
                         "error": {
                             "empty-message": "You must enter a message.",
-                            "max-characters": "Messages cannot contain more than 10,000 characters."
+                            "max-characters": "Your message cannot exceed 10,000 characters."
                         },
                         "success": "'<p>'Your comment has been sent to the Pix project team.'</p><p>'Thank you!'</p>'"
                     }
                 },
                 "information": {
-                    "data-usage": "'<p>'Pix processes the data from this area to manage and analyse the difficulty encountered and benefit from your feedback. You have rights over your data which can be exercised via '<a href=\"mailto:dpo@pix.fr\" class=\"link\">'dpo@pix.fr'</a>'.'</p><p><a href=\"https://pix.fr/dp-formulaire-signalement-epreuve\" target=\"_blank\" class=\"link\">'To find out more about protecting your data and your rights.'</a></p>'",
+                    "data-usage": "'<p>'Pix processes the data from this area to manage and analyse the difficulty encountered and benefit from your feedback. You have rights over your data which can be exercised via '<a href=\"mailto:dpo@pix.fr\" class=\"link\">'dpo@pix.fr'</a>'.'</p><p><a href=\"https://pix.org/en-gb/personal-data-reporting-test\" target=\"_blank\" class=\"link\">'To find out more about protecting your data and your rights.'</a></p>'",
```

J'ai un doute sur l'url : 
```suggestion
                    "data-usage": "'<p>'Pix processes the data from this area to manage and analyse the difficulty encountered and benefit from your feedback. You have rights over your data which can be exercised via '<a href=\"mailto:dpo@pix.fr\" class=\"link\">'dpo@pix.fr'</a>'.'</p><p><a href=\"https://pix.org/en-gb/personal-data-protection-policy\" target=\"_blank\" class=\"link\">'To find out more about protecting your data and your rights.'</a></p>'",
```
?

---

Apparemment c'est un problème lié aux pages "seules" que Clémentine a déjà remonté :
> J'ai une erreur 404 la page DP des signalements d'épreuve (le lien vers cette page est dispo uniquement sur Pix App en bas des épreuves, en dessous de "Signaler un problème") :
ORG-FR : pix.org/fr/dp-formulaire-signalement-epreuve/
ORG-EN : pix.org/en-gb/personal-data-reporting-test`

---

Oui ce n'est pas la même page apparemment et y'aurait un bug 🤔

---

en cours de correction ici : https://github.com/1024pix/pix-site/pull/261

---

corrigé !